### ISSUE TITLE
Extension .g.vcf.gz not supported for gvcf files in sample sheet

### ISSUE THREAD

**Issue by LysianeBouchard** [Ferlab-Ste-Justine/Post-processing-Pipeline#15]
### Description of the bug

This is an extension that is commun, so we should probably support it.
We probably simply need to tweak the regex in schema_input.json

Here is the input sample sheet:

Here is the error message observed:
```
ERROR ~ ERROR: Validation of 'input' file failed!

 -- Check '.nextflow.log' file for details
The following errors have been detected:

* -- Entry 1 - gvcf: Filename must have extension '.gvcf' or contain 'g' and end by 'vcf' (s3://cqdg-prod-file-import/test-datasets/remi_antoine_joron_internship2024/data/data/gvcf/NA12878.chr22.25000000_26000000.g.vcf.gz)
* -- Entry 2 - gvcf: Filename must have extension '.gvcf' or contain 'g' and end by 'vcf' (s3://cqdg-prod-file-import/test-datasets/remi_antoine_joron_internship2024/data/data/gvcf/NA12878.chr22.25000000_26000000.g.vcf.gz)
* -- Entry 3 - gvcf: Filename must have extension '.gvcf' or contain 'g' and end by 'vcf' (s3://cqdg-prod-file-import/test-datasets/remi_antoine_joron_internship2024/data/data/gvcf/NA12878.chr22.25000000_26000000.g.vcf.gz)
```

**Comment by FelixAntoineLeSieur** [comment]
yes, in #10 I changed the regex in schema_input.json to (g)?(enome.)?vcf(.gz)?$, which is probably even too permissive, since it would allow vcf also...

**Comment by FelixAntoineLeSieur** [comment]
This should be fixed in #19, I tested the regex with Damien. Accepts g.vcf, genome.vcf, refuses most other types, including .vcf, gavcf, etc...
